Description: Temporarily disable rkyv
 Nothing in Debian currently depends on chrno+rkyv, so temporarily disable to
 allow upgrading rkyv to 0.8.
 .
 Uses rkyv::ser::ScratchSpace which is removed in 0.8.
Author: Blair Noctis <ncts@debian.org>
Forwarded: not-needed
Last-Update: 2025-01-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -37,3 +37,3 @@
 
-[dependencies.bytecheck]
+[disabled.dependencies.bytecheck]
 version = "0.6.5"
@@ -49,3 +49,3 @@
 
-[dependencies.rkyv]
+[disabled.dependencies.rkyv]
 version = "0.7.18"
@@ -85,3 +85,3 @@
 
-[dev-dependencies.rkyv]
+[disabled.dev-dependencies.rkyv]
 version = "0.7.18"
@@ -98,5 +98,5 @@
 new_unchecked = []
-rkyv_validated = [
-    "rkyv",
-    "bytecheck",
-]
+#rkyv_validated = [
+#    "rkyv",
+#    "bytecheck",
+#]
